Formula/wsk.rb

```diff
@@ -0,0 +1,21 @@
+class Wsk < Formula
+  desc "OpenWhisk CLI"
+  homepage "https://console.ng.bluemix.net/openwhisk/cli"
+  url "https://openwhisk.ng.bluemix.net/cli/go/download/mac/amd64/OpenWhisk_CLI-mac.zip"
```

Comment:
The only version available is the commit id for the master branch, there are no tags nor other versions available in the project, or at least I can't find any. I guess the only option is https://github.com/openwhisk/openwhisk/archive/master.zip